src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -403,6 +403,38 @@ public ErrorCode InflateEnd()
                 return errC;
             }
 
+            /// <summary>
+            /// This function is equivalent to inflateEnd followed by inflateInit.
+            /// The stream will keep attributes that may have been set by inflateInit2.
+            /// </summary>
+            [SecurityCritical]
+            public ErrorCode InflateReset(int windowBits)
```

Comment:
I'm not 100% on the try/finally usage within this method - could use another look for sure.